ext/nnstreamer/tensor_filter/tensor_filter_tensorflow.cc

```diff
@@ -764,29 +766,36 @@ static GstTensorFilterFramework NNS_support_tensorflow = {
   .version = GST_TENSOR_FILTER_FRAMEWORK_V0,
   .open = tf_open,
   .close = tf_close,
+  {
+    .v0 = {
+      .name = filter_subplugin_tensorflow,
+      .allow_in_place = FALSE, /** @todo: support this to optimize performance later. */
+      .allocate_in_invoke = TRUE,
+      .run_without_model = FALSE,
+      .verify_model_path = FALSE,
+      .statistics = nullptr,
+      .invoke_NN = tf_run,
+      .getInputDimension = tf_getInputDim,
+      .getOutputDimension = tf_getOutputDim,
+      .setInputDimension = nullptr,
+      .destroyNotify = tf_destroyNotify,
+      .reloadModel = nullptr,
+      .checkAvailability = tf_checkAvailability,
+      .allocateInInvoke = nullptr, // TODO: what, it's allocate_in_invoke
```

Comment:
I'll remove this personal memo when I'm back home 😅

Reply:
> BTW @taos-ci is complaining about doxygen tags for components that this PR doesn't cover. This PR does not add any functions.

It is simply because the file you've touched has a function without proper ```@brief``` tag.